### ISSUE TITLE
[BUG] Multiplex People Screen: Extraneous Button on Search Field

### ISSUE THREAD

**Issue by jayna-SkylightDigital** [CDCgov/prime-simplereport#3847]
### Description
On the "People" screen, the search input field is attached to a blue "button" on the right. This button has a hover state but no label, and when clicked does not appear to have any impact on what is displayed on screen.

### Steps to reproduce

1. Go to the "People" screen
2. Begin typing in the search field. 
3. Click on the blue "button" next to the search field.
4. As you type a name into the Search field, the results automatically appear in the list on the screen.
5. Clicking on the blue button attached to the Search field does not affect the results displayed on screen.

### Expected behavior
As it currently exists, this button has no text label or function and provides no value to the user. 
This button should be removed.

### Additional context
Bug experienced on a MacBookPro running macOS Monterey Version 12.4
Chrome Version 102.0.5005.61 (Official Build) (x86_64)
Screenshot is attached.
![people-search](https://user-images.githubusercontent.com/100785345/171704088-f86c081e-6431-4c6e-92fa-daf4c607d02f.png)

**Comment by emmastephenson** [comment]
@kenieh Can you provide input on this? Should we take the blue button off entirely?

**Comment by kenieh** [comment]
I agree with @jayna-SkylightDigital , we should remove the blue button! That's the leanest thing we could do at this point.

I think some other implications that come to mind is if this feature—as it is without the button—is an accessible experience for screen readers or mobile friendly. I'm not sure about the accessibility aspect. In regards to mobile friendly, it's not but not sure if mobile is something we want to prioritize right now for the people page.

**Comment by lauraykerr** [comment]
@vz3 I remember you brought up a solution for this at some point — is there another ticket for this or was that a part of another piece of work?

**Comment by emyl3** [comment]
Blocked by https://github.com/CDCgov/prime-simplereport/issues/4695

**Comment by emyl3** [comment]
No longer blocked by #4695
